tools/generate-icons.js

```diff
@@ -122,7 +122,9 @@ export const ${iconName} = memo(forwardRef(function ${iconName}(props, ref) {
   const iconTypeDefs = iconNames.map(componentName => `export declare const ${componentName}: IconComponent`).join('\n')
 
   const iconsTypeDefs = `/* Start generated icons */
-export type IconComponent = React.ForwardRefExoticComponent<React.PropsWithoutRef<IconProps> & React.RefAttributes<SVGElement>>
+export type IconComponent = React.ForwardRefExoticComponent<
```

Comment:
This was churning from manual updates to `index.d.ts` due to our prettier config, so it should match now.